src/utils/BrambleExtensionLoader.js

```diff
@@ -60,7 +61,8 @@ define(function (require, exports, module) {
     var extraExtensions = [
         "brackets-cdn-suggestions",    // https://github.com/szdc/brackets-cdn-suggestions
         "ImageUrlCodeHints",
-        "HTMLHinter"
+        "HTMLHinter",
+        "Autosave"
```

Comment:
Why do we need it as an extra extension as well as a default extension? IMO it should just be a default extenstion